Marlin/Marlin_main.cpp

```diff
@@ -5419,7 +5438,14 @@ inline void gcode_M109() {
 
   } while (wait_for_heatup && TEMP_CONDITIONS);
 
-  if (wait_for_heatup) LCD_MESSAGEPGM(MSG_HEATING_COMPLETE);
+  if (wait_for_heatup) {
+    LCD_MESSAGEPGM(MSG_HEATING_COMPLETE);
+    #if ENABLED(LED_STRIP)
+      // Heating is complete, turn the LED Strip to all white
+      rgb_to_led_strip(255, 255, 255);
+    #endif
```

Comment:
If generalized for all the LED options, then this will be:
```cpp
    #if ENABLED(PRINTER_EVENT_LEDS)
      // Heating is complete, turn the LED Strip to all white
      set_led_color(255, 255, 255);
    #endif
```